### PR TITLE
Update README.md :books:

### PR DESCRIPTION
**Pull request by Freymaurer**
Just update README.md to provide a cleaner entrypoint 📚

### CODE REVIEW

File: docs/ARCtrl_Design.md
Comment (HLWeil) [inline docs/ARCtrl_Design.md:1]: Maybe rename to "Design_Principles.md"?

File: docs/ARCtrl_Design.md [hunk 1]
```diff
@@ -0,0 +1,292 @@
+- [Jargon/Nomenclature](#jargonnomenclature)
```
Comment (HLWeil) [inline docs/ARCtrl_Design.md:1]: Include header above toc

File: README.md [hunk 2]
```diff
@@ -1,280 +1,53 @@
...
 
-- [ARCtrl](#arctrl)
-  - [Jargon/Nomenclature](#jargonnomenclature)
-  - [API Design](#api-design)
-  - [Top level overview](#top-level-overview)
-  - [Stack](#stack)
-- [Libraries](#libraries)
-  - [Design choices](#design-choices)
-    - [Fable compatibility as top priority](#fable-compatibility-as-top-priority)
+| Version | Downloads |
```
Comment (HLWeil) [inline README.md:6]: Maybe we could include a "Tests/Codecov" badge?

File: README.md [hunk 3]
```diff
@@ -1,280 +1,53 @@
...
-style fsspread fill:#39B539,stroke:#007B00,stroke-width:2px,color: black
-style isax fill:#39B539,stroke:#007B00,stroke-width:2px,color: black
-style isaa fill:#39B539,stroke:#007B00,stroke-width:2px,color: black
-style isaj fill:#39B539,stroke:#007B00,stroke-width:2px,color: black
-style arc fill:#39B539,stroke:#007B00,stroke-width:2px,color: black
-style fs fill:#39B539,stroke:#007B00,stroke-width:2px,color: black
-style cwl fill:#39B539,stroke:#007B00,stroke-width:2px,color: black
-style thoth fill:#39B539,stroke:#007B00,stroke-width:2px,color: black
+```bash
+<PackageReference Include="ARCtrl" Version="1.0.0-alpha9" />
```
Comment (HLWeil) [inline README.md:22]: Directly update to beta.1?

File: README.md [hunk 4]
```diff
@@ -1,280 +1,53 @@
...
 
+```bash
+npm i @nfdi4plants/arctrl
 ```
 
-# Libraries
+## Docs
 
-Mockups for each library are in the `ARC` folder. They contain API/Domain stubs to draft and test the new library designs.
+Currently we provide some documentation in form of markdown fiels in the `/docs` folder of this repository!
```
Comment (HLWeil) [inline README.md:33]: spelling error: "fiels"

File: README.md [hunk 5]
```diff
@@ -1,280 +1,53 @@
...
 
-## Design choices
+[Check it out!](/docs)
```
Comment (HLWeil) [inline README.md:35]: Maybe directly include the links to the docs here?

**Review by HLWeil — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (Freymaurer) [reply README.md:35]: it is? 😅

File: README.md (hunk 2 above)
Comment (Freymaurer) [reply README.md:6]: Yeah thats a bit of work i am not willing to do for this PR. For more information see here:

- https://github.com/TheAngryByrd/MiniScaffold/blob/755f63a4b8daf70f10266b1e124d7c39fffcaac7/Content/Library/build/build.fs#L443
- https://github.com/SteveGilham/altcover

**Pull request merged**